addons/purchase/security/ir.model.access.csv

```diff
@@ -50,6 +49,6 @@ access_account_account_purchase_manager,account.account purchase manager,account
 access_account_journal_purchase_manager,account.journal purchase manager,account.model_account_journal,purchase.group_purchase_manager,1,0,0,0
 access_stock_location_purchase_manager,stock.location purchase manager,stock.model_stock_location,purchase.group_purchase_manager,1,0,0,0
 access_stock_warehouse_orderpoint_manager,stock.warehouse.orderpoint,stock.model_stock_warehouse_orderpoint,purchase.group_purchase_manager,1,0,0,0
-access_stock_warehouse_orderpoint_manager,stock.warehouse.orderpoint,stock.model_stock_warehouse_orderpoint,group_purchase_user,1,0,0,0
+access_stock_warehouse_orderpoint_user,stock.warehouse.orderpoint,stock.model_stock_warehouse_orderpoint,group_purchase_user,1,0,0,0

```

Comment:
It's duplicated from https://github.com/odoo/odoo/blob/1d34e269dac2e0cafd85ad91a6240569b5b7282c/addons/purchase/security/ir.model.access.csv#L52-L53